crest/Assets/Crest/Crest/Scripts/LodData/Shadows/RegisterShadowInput.cs

```diff
@@ -21,5 +21,11 @@ public class RegisterShadowInput : RegisterLodDataInput<LodDataMgrShadow>
         protected override string ShaderPrefix => "Crest/Inputs/Shadows";
 
         protected override bool FollowHorizontalMotion => false;
+
+        protected override bool FeatureEnabled(OceanRenderer ocean) => ocean.CreateShadowData;
+        protected override string FeatureDisabledErrorMessage => "<i>Create Shadow Data</i> must be enabled on the OceanRenderer component.";
+
+        protected override string RequiredShaderKeyword => LodDataMgrShadow.MATERIAL_KEYWORD;
+        protected override string KeywordMissingErrorMessage => LodDataMgrShadow.MATERIAL_KEYWORD_MISSING_ERROR;
```

Comment:
Will need to be wrapped in *UNITY_EDITOR* so it can build. Same goes with the others.